CatFoodCalc/presets.py

```diff
@@ -0,0 +1,27 @@
+age = {
+    1: "kitten under 4 mo",
+    2: "kitten under 9 mo",
+    3: "kitten under 12 mo",
+    4: "adult cat",
+    5: "pregnant cat",
+    6: "lactating cat (under 3 kittens)",
+    7: "lactating cat (3+ kittens)",
+    8: "senior cat"
+}
+
+activity = {
+    "indoor": "neutered, inactive and indoor cats",
```

Comment:
Keywords in this scenario will make using your dictionary easier, your code more understandable and they also allow constructions like the one I used in Cat.get_kcal_per_kg_by_activity(), line 30